functions/concepts/filesystem.rb

```diff
@@ -0,0 +1,24 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# [START functions_concepts_filesystem]
+require "functions_framework"
+require "cgi"
+
+FunctionsFramework.http "concepts-filesystem" do |_request|
```

Comment:
Please use underscores rather than hyphens in function names. (Yes, I know some of my existing samples still have hyphens. They were written before I learned about the restriction, and they do need to be fixed.)